API/testrunner/devices/device_base.py

```diff
@@ -64,7 +64,10 @@ def login(self):
                 # Press enters to start the serial communication and
                 # go to the test folder because some tests require resources.
                 self.channel.exec_command('\n\n')
-                self.channel.exec_command('cd /rom')
+                if self.device == 'artik053':
+                    self.channel.exec_command('cd /rom')
```

Comment:
Please modify the https://github.com/Samsung/js-remote-test/blob/master/API/resources/configs/tizenrt-iotjs.config#L248 to `test` to eliminate the `if` statement.